test/integration/targets/luks_device/tasks/tests/create-destroy.yml

```diff
@@ -14,13 +14,29 @@
     keyfile: "{{ role_path }}/files/keyfile1"
   become: yes
   register: create
+- name: Create with keysize
```

Comment:
Don't put other tests between a test and its idempotent variant. That breaks the existing test logic and in this case won't even run your test, since the device is already there, so the command will never be executed.

Reply:
(Put it in a new file `options.yml` or so.)